esp/templates/program/modules/programprintables/completion_certificate.tex

```diff
@@ -15,7 +15,12 @@
 
 \vspace{0.25in}
 
-{{ prog.program_type }} courses are taught by volunteers, primarily MIT students.  The content and difficulty of the subjects varies significantly, from non-academic to college-level.  We are not an accredited institution and cannot issue high-school or college credit for any class.  However, we leave it to school administrators to decide whether ESP courses can be counted as substitutes for high-school classes.  You may view the teachers' course descriptions on the attached pages.  If you would like to talk to an administrator or teacher about the content of a particular course, please e-mail us at {\tt esp@mit.edu} or call us at (617) 253-4882. \\
+{{ prog.program_type }} courses are taught by volunteers, primarily {{ settings.INSTITUTION_NAME }} students.
+The content and difficulty of the subjects varies significantly, from non-academic to college-level.
+We are not an accredited institution and cannot issue high-school or college credit for any class.
+However, we leave it to school administrators to decide whether {{ settings.ORGANIZATION_SHORT_NAME }} courses can be counted as substitutes for high-school classes.
+You may view the teachers' course descriptions on the attached pages.
+If you would like to talk to an administrator or teacher about the content of a particular course, please email us at {\tt esp@mit.edu} or call us at (617) 253-4882. \\
```

Comment:
Shouldn't this be generalized?

Reply:
(and below)

Reply:
yeah, the phone number isn't generalizable, but there's always template overrides